core/src/banking_stage/transaction_scheduler/receive_and_buffer.rs

```diff
@@ -336,7 +337,7 @@ impl ReceiveAndBuffer for TransactionViewReceiveAndBuffer {
             }
         }
 
-        while start.elapsed() < TIMEOUT {
+        while start.elapsed() < TIMEOUT && num_received < PACKET_BURST_LIMIT {
```

Comment:
note that SV can still send down more than 1k packets in a batch, but this will at least early out right after, so still an improvement.

There is a subsequent change that modifies SV sending behavior to better align with this limit

Reply:
yeah to be clear it will NOT drop the additional packets. If it receives a message with 5k packets it will still process all 5k. it will just not keep going after that message.